src/resources/views/crud/columns/select_dependency.blade.php

```diff
@@ -0,0 +1,70 @@
+{{-- single relationships (1-1, 1-n) --}}
```

Comment:
Hmmm there is no such `select_dependency` field so... let's move this code to `checklist_dependency` please 🙏

Reply:
Should I remove `select_dependency` column as that is NOT needed. Please suggest?

Reply:
Removed `select_dependency` & merge code in `checklist_dependency`. let me know